playbooks/prepare_host.yaml

```diff
@@ -48,6 +48,25 @@
           owner: stack
           group: stack
 
+  # To ensure that our CentOS users are actually using Stream.
+  # TripleO Master only support Stream now.
+  - name: Migrate to CentOS Stream if not done already
+    when:
+      - ansible_facts.distribution == 'CentOS'
+      - ansible_facts.distribution_release != 'Stream'
```

Comment:
Even after a reboot of my now CentOS Stream node, I still get 
```
"ansible_distribution": "CentOS",                                                                                                                               
"ansible_distribution_file_parsed": true,
"ansible_distribution_file_path": "/etc/redhat-release",
"ansible_distribution_file_variety": "RedHat",
"ansible_distribution_major_version": "8",
"ansible_distribution_release": "NA",
"ansible_distribution_version": "8",
```

Perhaps we should parse `/etc/redhat-release` instead?
```
[centos@manila-cloud ~]$ cat /etc/redhat-release 
CentOS Stream release 8
```

Reply:
yeah that fact was integrated in the latest version of Ansible.
Honestly, I wish we could avoid that PR, I'll mark it as WIP for now.

Once we can get CentOS 8 stream on VEXXHOST, we wouldn't need that.